tests/jballerina-unit-test/src/test/resources/test-src/query/string-query-expression.bal

```diff
@@ -41,6 +41,64 @@ function testSimpleQueryExprForStringResult() returns string {
     return outputNameString;
 }
 
+type Book record {|
+    string title;
+    string author;
+|};
+
+class BookGenerator {
+    int i = 0;
+
+    public isolated function next() returns record {|Book value;|}|error? {
+        self.i += 1;
+        if (self.i < 0) {
+            return error("Error");
+        } else if (self.i >= 3) {
+            return ();
+        }
+        return {
+            value: {
+                title: "Title " + self.i.toString(), author: "Author " + self.i.toString()
+            }
+        };
+    }
+}
+
+function testSimpleQueryExprForStringResult2() {
+    error? e = trap simpleQueryExprForStringResult();
+    assertFalse(e is error);
+
+    e = trap simpleQueryExprForStringResult2();
+    assertFalse(e is error);
+}
+
+function simpleQueryExprForStringResult() returns error? {
+    string:Char chr = "a";
+    BookGenerator bookGenerator = new ();
+    stream<Book, error?> bookStream = new (bookGenerator);
+
```

Comment:
```suggestion
```

Reply:
These are intentionally put to improve the readability of the code.

```
test function () {
  < var declaration>
  empty line
  <query declaration>
  empty line
  <assertions>
}
```

Wdyt?